### PR TITLE
README: Add notice about me not using it anymore

### PR DESCRIPTION
**Pull request by ldoktor**
I moved towards using jenkins-job-builder instead of this custom code.
Feel free to inspire/fork, but I'd suggest doing the same as me...

Signed-off-by: Lukáš Doktor ldoktor@redhat.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # JenkinsMaster
 
 Tool to create and deploy multiple variants of jenkins jobs defined by YAML file and XML templates.
+
+__I don't have any use for this tool anymore as I use jenkins-job-builder instead.__
```
Comment (ldoktor) [inline README.md:5]: Acked-by: Lukáš Doktor ldoktor@redhat.com

**Review by ldoktor — COMMENTED** [review]
Good to go

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
 # JenkinsMaster
 
 Tool to create and deploy multiple variants of jenkins jobs defined by YAML file and XML templates.
```
Comment (apahim) [inline README.md:3]: Ok

**Review by apahim — APPROVED** [review]

**Review by apahim — COMMENTED** [review]
Ack

**Review by apahim — APPROVED** [review]
Ack

**Pull request merged**